src/Helper/Navigation/PluginManager.php

```diff
@@ -81,6 +48,7 @@ public function __construct($configOrContainerInstance = null, array $v3config =
             $instance->setServiceLocator($this->creationContext);
         };
 
-        parent::__construct($configOrContainerInstance, $v3config);
+        $this->aliases   = ConfigProvider::defaultViewHelperAliases();
```

Comment:
I haven't checked this due to laziness, but the aliases which were registered must not change in any way. Juggling between aliases/factories introduces BC breaks as we learned here:

https://github.com/laminas/laminas-mvc/issues/99

If you can confirm that the `factories` and the `aliases` are 100% the same as in the previous version, we're fine here.

Reply:
All of the factories and aliases *were* the same, but since adding extra tests, they've uncovered issues with the original config such as a factory for `'somenormalisedname' =>  InvokableFactory::class` which of course blows up if you attempt to fetch by that name. I've amended these so that they *alias* the correct target - effectively a bug fix.

Considering they could never have worked in the first place, it might be a good idea to just remove them completely instead.